__tests__/app.test.js

```diff
@@ -4,6 +4,7 @@ const seed = require("../db/seeds/seed");
 const data = require("../db/data/test-data");
 const db = require("../db/connection.js");
 const endpointsDoc = require("../endpoints.json");
+const sorted = require("jest-sorted");
```

Comment:
You don't need to import this, you should have set it up in your package.json correctly